app/controllers/merchants/invoices_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Merchants::InvoicesController < ApplicationController
+
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @merchant_invoices = @merchant.items.merchant_invoices
```

Comment:
Is this technically a law of demeter violation?

Reply:
I think you're right...I can refactor this.